android/build.gradle

```diff
@@ -132,6 +134,13 @@ android {
       } else {
         java.srcDirs += ["src/oldarch"]
       }
+
+      // TextLayoutManager
```

Comment:
This is not very informative

Reply:
I followed the convention from https://github.com/software-mansion/react-native-reanimated/blob/82ecdc996953cc35001f3f039112290649b37cdf/packages/react-native-reanimated/android/build.gradle#L350 🤷‍♂️